### PR TITLE
Ignore banner prompts during installation

### PR DESCRIPTION
**Pull request by edewata**
`pkispawn` has been modified to ignore banner prompts during installation. New tests have been added to verify installation with banner.

https://bugzilla.redhat.com/show_bug.cgi?id=2069448

### CODE REVIEW

**Review by ckelleyRH — COMMENTED** [review]
LGTM but will leave it to @snehaveeranki1 or @ladycfu approve.

File: base/server/examples/banner/banner.txt
Comment (ladycfu) [inline base/server/examples/banner/banner.txt:5]: A bit ominous ;-).  How about the following instead?
"Access to this service is restricted to those individuals with specific permissions.  Any attempts to gain access, authorized or not, and all operations that follow, are audited and could be used against you in the court of law."

**Review by ladycfu — APPROVED** [review]
In the interest of time, this review doesn't include the CI (assuming it passes).
I only have one very minor suggestion to the example banner text.
I'll leave it to you on whether to adopt it or not.

File: base/server/examples/banner/banner.txt
Comment (edewata) [reply base/server/examples/banner/banner.txt]: Sure, I updated the PR and these pages too:
https://github.com/dogtagpki/pki/wiki/Access-Banner
https://github.com/dogtagpki/pki/wiki/PKI-10.4-Access-Banner-Design
https://github.com/dogtagpki/pki/wiki/PKI-Server-Banner-CLI
https://github.com/dogtagpki/pki/wiki/PKI-CLI-Options
https://github.com/dogtagpki/pki/wiki/PKI-Console

**Comment by edewata** [comment]
@ladycfu Thanks! I've updated the example. I'll merge once the CI completes.

**Pull request merged**
